docs/user/gcp/install_upi.md

```diff
@@ -252,8 +283,8 @@ export CLUSTER_PUBLIC_IP=$(gcloud compute addresses describe ${INFRA_ID}-cluster
 ```
 
 ## Add DNS entries
-The templates do not create DNS entries due to limitations of Deployment
-Manager, so we must create them manually.
+The templates do not create DNS entries due to limitations of Deployment Manager, so we must create them manually.
+If you are installing into a Shared VPC, use the `--account` and `--project` parameters to perform these actions in the host project.
```

Comment:
separate out the XPN case to a different paragraph.

Reply:
Updated with 998a518a17e54a792bd52cf31b217775eb20f5a3